conformance/chaos/actor.go

```diff
@@ -230,3 +234,21 @@ func (a Actor) MutateState(rt runtime.Runtime, args *MutateStateArgs) *adt.Empty
 	}
 	return nil
 }
+
+// AbortArgs are the arguments to the abort method, specifying the exit code to
+// (optionally) abort with and the message.
+type AbortArgs struct {
+	Code    exitcode.ExitCode
+	NoCode  bool
```

Comment:
Can we call this `Panic` instead?

Reply:
We decided `Uncontrolled` was less implementation specific.